module_bindings/dripline_core/run_simple_service_pybind.hh

```diff
@@ -0,0 +1,12 @@
+#include "run_simple_service.hh"
+#include "pybind11/pybind11.h"
+
+namespace dripline_pybind {
+
+  void export_run_simple_service(pybind11::module& mod){
+    pybind11::class_<dripline::run_simple_service, std::shared_ptr<dripline::run_simple_service> >(mod, "run_simple_service")
```

Comment:
same comment on `()`, also applies to template arguments `<>`

Reply:
please also add a class doc string, should just be another string argument; I'm not sure how consistently we've applied them, but we would like to have them on basically everything we choose to bind.

Reply:
So something like _void export_run_simple_service( pybind11::module& mod, std::string doc )_